pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -474,13 +474,14 @@ class Build {
     // Temurin remote jck trigger
     def remoteTriggerJckTests(String platform) {
         def jdkVersion = getJavaVersionNumber()
-        //def sdkUrl="https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk${jdkVersion}-pipeline/${env.BUILD_NUMBER}/"
-        def filter = "*.tar.gz"
+        def jdkUrl = "${env.BUILD_URL}/artifact/workspace/target/${buildConfig.TARGET_OS}/${buildConfig.ARCHITECTURE}/${buildConfig.VARIANT}/*-jdk_*"
```

Comment:
Is this URL correct? I feels like this is the url for pipeline build not jdk build?

Reply:
thank you @sophia-guo you're correct